### PR TITLE
keystone: update ldap config parameters in template

### PR DESCRIPTION
**Pull request by sidthe**

### CODE REVIEW

File: chef/cookbooks/keystone/attributes/default.rb [hunk 1]
```diff
@@ -92,19 +92,26 @@
...
 default[:keystone][:ldap][:group_member_attribute] = "member"
-default[:keystone][:ldap][:group_attribute_ignore] = ""
-default[:keystone][:ldap][:group_allow_create] = true
-default[:keystone][:ldap][:group_allow_update] = true
-default[:keystone][:ldap][:group_allow_delete] = true
-default[:keystone][:ldap][:use_pool] = true
 default[:keystone][:ldap][:group_members_are_ids] = false
+default[:keystone][:ldap][:group_additional_attribute_mapping] = ""
+default[:keystone][:ldap][:group_ad_nesting] = false
+default[:keystone][:ldap][:use_pool] = true
```
Comment (Itxaka) [inline chef/cookbooks/keystone/attributes/default.rb]: This is the default value already provided in the default config, if we are not gonna change it, doesn't it make sense to just remove it?

After all, we want to only add keys that we know we are overriding while letting the default values stay.

File: chef/cookbooks/keystone/attributes/default.rb [hunk 2]
```diff
@@ -92,19 +92,26 @@
...
+default[:keystone][:ldap][:pool_size] = 10
+default[:keystone][:ldap][:pool_retry_max] = 3
```
Comment (Itxaka) [inline chef/cookbooks/keystone/attributes/default.rb]: from here downwards, all keys are already in their default values as commented above. It migth make more sense to just remove them unless we are gonna override them.

File: chef/cookbooks/keystone/attributes/default.rb (hunk 1 above)
Comment (sidthe) [inline chef/cookbooks/keystone/attributes/default.rb]: @Itxaka for LDAP the rules is to expose all parameters to the user, this is what the card is all about

**Comment by sidthe** [comment]
[(2) Ensure new Keystone LDAP options are in schema](https://trello.com/c/5WlLyj70/396-2-ensure-new-keystone-ldap-options-are-in-schema)

File: chef/cookbooks/keystone/attributes/default.rb (hunk 2 above)
Comment (sidthe) [inline chef/cookbooks/keystone/attributes/default.rb]: @Itxaka as written above. For LDAP we expose everything to the user.

File: chef/cookbooks/keystone/attributes/default.rb (hunk 1 above)
Comment (Itxaka) [inline chef/cookbooks/keystone/attributes/default.rb]: Looks good then :D

**Pull request merged**